lucien-ios/Models/Comic.swift

```diff
@@ -8,133 +8,143 @@
 
 import Foundation
 
-struct Comic {
+protocol Comic: Decodable {
 
+    var comicID: String { get set }
+    var comicTitle: String { get set }
```

Comment:
These values should probably be `get` only since it's better practice to create a new instance rather than mutate the current instance